client/my-sites/upgrades/controller.jsx

```diff
@@ -161,6 +171,50 @@ module.exports = {
 		);
 	},
 
+	selectPlan: function( context ) {
+		const basePath = route.sectionify( context.path );
+		const CartData = require( 'components/data/cart' );
+		const Plans = require( './plans' );
+		const domain = context.params.registerDomain;
+
+		const onGoBack = () => {
+			page( '/domains/add/' + sites.getSelectedSite().slug );
+		};
+
+		analytics.pageView.record( basePath, 'Domain Search > Domain Registration > Select Plan' );
```

Comment:
Updated (but only the new methods) with `<PageViewTracker />`. Thanks for bringing this up, I wasn't aware of that component 😊